pkg/client/src/app/Constants.ts

```diff
@@ -19,9 +19,11 @@ export interface IEnvVars {
   KEYCLOAK_REALM: string;
   KEYCLOAK_CLIENT_ID: string;
   KEYCLOAK_SERVER_URL: string;
+  UI_INGRESS_PROXY_BODY_SIZE: string;
 }
 export const ENV: IEnvVars = window["_env"] || {};
 export const isAuthRequired = ENV.AUTH_REQUIRED !== "false";
+export const uploadLimit = ENV.UI_INGRESS_PROXY_BODY_SIZE || "500m";
```

Comment:
I think MB or MiB is preferred.wdty?
@ibolton336

Reply:
@amastbau https://kubernetes.github.io/ingress-nginx/user-guide/nginx-configuration/annotations/#custom-max-body-size Going off of the format used from the ingress annotation here. It seems that they use 'm' to indicate megabytes & not sure there is a way to change it.

Reply:
@jortel @fbladilo